src/Sidebar.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from "react";
+import homeicon from "./Images/home-icon.png";
+import search from "./Images/search.png";
+import detail from "./Images/detail-icon.png";
+import baropener from "./Images/bar-opener.png";
+import profile from "./Images/profile logo.png";
+
+export default function Sidebar() {
+  const [sidebar, setsidebar] = useState(true);
```

Comment:
The convention for setters is to follow camelCase. So setsidebar should be setSidebar